Makefile

```diff
@@ -170,6 +176,16 @@ IMG = quay.io/crcont/crc-e2e:$(CRC_E2E_IMG_VERSION)
 containerized_e2e: clean
 	$(CONTAINER_RUNTIME) build -t $(IMG) -f images/build-e2e/Dockerfile .
 
+#  Build the container image for integration
+.PHONY: containerized_integration
+containerized_integration:
+ifndef CRC_INTEGRATION_IMG_VERSION
+CRC_INTEGRATION_IMG_VERSION=v$(CRC_VERSION)-$(COMMIT_SHA)
+endif
+IMG = quay.io/crcont/crc-integration:$(CRC_INTEGRATION_IMG_VERSION)
```

Comment:
Agree if both images contains same version tag but it would be difficult if you want to use different tag for different target on a single make run. If those target suppose to run separately and not interdependent then having same variable make sense.

Reply:
yeah totally agree, as I added to the PR the idea is have a following PR were probably those images can be merged into one..having something like:

`quay.io/crcont/crc-test:$CRC_VERSION-$OS_PLATFORM`

but for the time being I would prefer to finish this first iteration and then move ahead with the single image for testing

Reply:
WDYT?

Reply:
I was not suggesting single image which is more work, just single `VERSION` variable.